### DISCUSSION TITLE
能否整理下在检查更新时不写入已失效插件

### DISCUSSION THREAD

**Discussion by mnwgkj** [MoeNetwork/Tieba-Cloud-Sign#202] [想法]
新人不知道哪些插件失效哪些能用

**Comment by BANKA2017** [comment]
云签对绝大多数插件都是向后兼容的，官方无法得知插件本身的功能是否失效

**Comment by n0099** [comment]
https://github.com/MoeNetwork/Tieba-Cloud-Sign/discussions/204#discussioncomment-3613797
